components/EventCard/EventCard.component.js

```diff
@@ -24,17 +24,23 @@ const EventCard = (props) => {
                 <Text style={styles.title}> {props.title} </Text>
                 <Image style={styles.image} resizeMode="contain" source={props.source}/>
                 <Text style={styles.date}> {props.date} </Text>
+
                 <Text> {props.link} </Text>
             </TouchableOpacity>
+            
             <Modal animationType="slide"
                 transparent={true}
-                visible={isModalVisible}>
+                visible={isModalVisible}
+                onBackdropPress = { () => this.setState({isVisible:false})}
```

Comment:
This prop doesn't exist for Modal. What were you trying to do?
https://reactnative.dev/docs/modal

Reply:
Yeah, I was tying to do an exit out of the modal by clicking outside of the modal popup.